### DISCUSSION TITLE
Chip nRF24L01+ connection

### DISCUSSION THREAD

**Discussion by andrerdrummond** [tbnobody/OpenDTU#373] [Q&A] [answered]
I'm having problems with the CHIP connection, it's unstable. How can I fix this? Someone can help me?
![CHIP OFF](https://user-images.githubusercontent.com/11406518/203325036-5497de6b-fd7d-459d-867f-1f87b78269a9.JPG)
![CHIP ON](https://user-images.githubusercontent.com/11406518/203325040-cd9226d5-2251-4c96-90c6-b4be4a3659ca.JPG)

**Comment by andrerdrummond** [comment]
![image](https://user-images.githubusercontent.com/11406518/203602165-9b6ca9e6-8cce-4852-859a-0496f8db45f5.png)

**Reply by tbnobody** [reply]
it seems that your wiring is not correct. but most important. is your inverter really a MI-1500??? As you've already read in the README.md, only HM models are supported. MI models are using a different  protocol.

**Reply by andrerdrummond** [reply]
It was working a few versions ago, since the last one it started this problem. And Yes, it's a MI-1500, seems to work with the latest protocol. I've already opened an issue and you made some changes at serial number validation at HM_4CH.cpp. I bought more nrf24L01+ to test if the problem persist.

**Reply by andrerdrummond** [reply]
It's a MI-1500 Serial Number: 1062734XXXXX

**Reply by andrerdrummond** [reply]
![image](https://user-images.githubusercontent.com/11406518/203615674-cdfeee1e-daf6-4f85-8db4-b5d2d495cdbc.png)

**Reply by tbnobody** [reply]
oh sorry. I didn't remember for whom I changed the detection method...

**Comment by andrerdrummond** [comment, marked as answer]
With a PCB I fixed the problem. The wires that i used wasn't connecting the chip correctly.
